src/components/frame/v5/pages/UserCryptoToFiatPage/context/CryptoToFiatContextProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { type PropsWithChildren, useEffect, useMemo } from 'react';
+
+import { useCheckKycStatusMutation } from '~gql';
+
+import { CryptoToFiatContext } from './CryptoToFiatContext.ts';
+
+const CryptoToFiatContextProvider: React.FC<PropsWithChildren> = ({
+  children,
+}) => {
+  const [getKycStatusData, { loading, data }] = useCheckKycStatusMutation();
+
+  useEffect(() => {
+    getKycStatusData();
+  }, [getKycStatusData]);
+
+  const value = useMemo(
+    () => ({
+      getKycStatusData,
+      kycStatusData: data?.bridgeXYZMutation,
+      bankAccountData: data?.bridgeXYZMutation?.bankAccount,
```

Comment:
This feels like we're duplicating data since bank account can be accessed under kycStatusData

Reply:
I exported it like that because I just want the usage to be simple like:
```js
const { bankAccountData } = useCryptoToFiat();

console.log(bankAccountData);

// versus
const { kycStatusData } = useCryptoToFiat();

console.log(kycStatusData?.bankAccount);
```

Is this all right?